game-frontend/src/useWebsocketData.js

```diff
@@ -6,6 +6,9 @@ function useWebsocketData() {
   const [gamePosition, setGamePosition] = useState(null);
   useEffect(() => {
     const socket = new WebSocket(API_URL);
+    socket.onopen = function(event) {
+      socket.send('Hello server!');
+    };
```

Comment:
This should be on a different branch imo. @sir-marc ?

Reply:
Was just to test the message event on the backend, will get removed anyway later.